pjsip/src/pjsua-lib/pjsua_aud.c

```diff
@@ -2155,6 +2155,12 @@ static void close_snd_dev(void)
 {
     pj_log_push_indent();
 
+    /* Cancel sound device idle timer. */
+    if (0 && pjsua_var.snd_idle_timer.id) {
```

Comment:
if 0?

Reply:
Was generating log to show the problem, but then forgot to revert it.